### ISSUE TITLE
Typed message lost when switching from mobile to desktop layout

### ISSUE THREAD

**Issue by Siltaar** [NodeBB/NodeBB#6353]
A problem that looks like related to the responsive design.
Can be reproduced with Firefox 58 and current TOR Browser (Firefox 52).
NodeBB v1.7.3

**Steps to reproduce :**
1. Zoom your browser until you get the mobile layout.
2. Answer a subject (you'll get a full width editor)
3. Type some message
4. Zoom out your browser until you get the desktop layout

**Result**
You get an empty desktop sized editor (using half the available space, the other half behing shared with the preview).

**Expected behavior**
I would have expect to still have my ongoing message in the editor.
Moreover, I can't get my message back zooming-in again.

**Other way to reproduce the bug**

- Resize your browser window until you get the mobile layout. Type your message. Resize your browser window to switch to the desktop layout. Message lost.
- So, basically : get your browser half-windowed, type the message, press F11 to get the browser full screen, message lost (retype F11 to return to the mobile layout : message still lost).

**Comment by julianlam** [comment]
Hi @Siltaar ! Thanks for submitting this bug report, it looks to be a valid bug that really ought to be fixed :smile:

**Comment by julianlam** [comment]
Actually, as it turns out, I no longer experience this issue... let me know if it still crops up. Tried zoom in/out, and browser resize + F11, as well as standard browser resize. No dice.

**Comment by barisusakli** [comment]
Not gonna open a new issue for this but I can reproduce it on latest firefox. Here is what happens.

![bug](https://user-images.githubusercontent.com/2583394/46491212-1987e580-c7d8-11e8-95d3-b361d7238ba2.gif)

**Comment by barisusakli** [comment]
On chrome the composer just goes back to desktop layout

**Comment by julianlam** [comment]
That... is bizarre...
